scripts/openzfs-tracking.sh

```diff
@@ -84,22 +98,102 @@ while getopts 'hd:e:' OPTION; do
 done
 
 cat << EOF
+<!DOCTYPE html>
 <html>
 <head>
 <title>OpenZFS Tracking</title>
 <meta name="keyword" content="zfs, linux"/>
+<meta http-equiv="Content-type" content="text/html; charset=utf-8">
+<script		  src="https://code.jquery.com/jquery-1.12.4.min.js"
+			  integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="
+			  crossorigin="anonymous"></script>
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css">
+<script type="text/javascript" language="javascript" src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
+
+<script type="text/javascript">
+\$(document).ready(function() {
+	\$('#maintable').DataTable( {
+		 "lengthMenu": [[50, 100, 500, -1], [50, 100, 500, "All"]],
+		 "order": [],
+		initComplete: function () {
+				var column = this.api().column(4);
+				var select = \$('<select><option value=""></option></select>')
+					.appendTo( \$(column.header()).empty() )
+					.on( 'change', function () {
+						var val = $.fn.dataTable.util.escapeRegex(
+							\$(this).val()
+						);
+
+						column
+							.search( val ? '^'+val+'$' : '', true, false )
+							.draw();
+					} );
+
+				column.data().unique().sort().each( function ( d, j ) {
+					select.append( '<option value="'+d+'">'+d+'</option>' )
+				} );
+		}
+	} );
+} );
+</script>
+
+<style>
+#maindiv {
+	width:90%;
+	display: table;
+	margin: 0 auto;
+}
+#intro {
+	text-align:center;
+	padding:20px;	
+}
+#f_date {
+	text-align:right;
+}
+#maintable {
+	text-align:center;
+	border:0px;	
+}
+.st_appl {
+	background:$STATUS_APPLIED_COLOR !important;
+}
+.st_exc {
+	background:$STATUS_EXCEPTION_COLOR !important;
+}
+.st_mis {
+	background:$STATUS_MISSING_COLOR !important;
+}
+.st_pr {
+	background:$STATUS_PR_COLOR !important;
+}
+.st_na {
+	background:$STATUS_NONAPPLICABLE_COLOR !important;
+}
+.td_text {
+	text-align:left;
+	min-width:60%;
+}
+</style>
+
 </head>
 <body>
-<table align="center" width="80%" border="0">
-<tr bgcolor='#aaaaaa'>
-  <th colspan='4'>OpenZFS Commit Tracking</th>
-</tr>
-<tr bgcolor='#dddddd'>
+<h1 align='center'>OpenZFS Commit Tracking</h1>
+<div id="intro">
+This page is updated regularly and shows a list of OpenZFS commits and their status in regard to the ZFS on Linux master branch. See <a href="https://github.com/zfsonlinux/zfs/wiki/OpenZFS-Patches">wiki</a> for more information about OpenZFS patches.
+</div>
+<div id='maindiv'>
+<div id="f_date">Last Update: $DATE by <a href="https://github.com/zfsonlinux/zfs-buildbot/blob/master/scripts/openzfs-tracking.sh">openzfs-tracking.sh</a></div>
+<table id="maintable" class="display">
+<thead>
+<tr>
   <th>OpenZFS Issue</th>
   <th>OpenZFS Commit</th>
   <th>Linux Commit</th>
-  <th>Description</th>
+  <th class='td_text'>Description</th>
+  <th>Status</th>
```

Comment:
I can't say that it must be hidden, it's easier personally for me to see the status. We can add any comment to this column later.

I can hide it and add a legend, if it's better. Yes, it's used for filter.